gzipi/cli.py

```diff
@@ -214,7 +218,7 @@ def _repack_subparser(subparsers):
     parser.add_argument('--field', required=False, default=lib.DEFAULT_JSON_FIELD,
                         help='The name of key field to use for JSON format.')
     parser.add_argument('--chunk-size', required=False, default=lib.DEFAULT_CHUNK_SIZE,
-                        help='The number of lines to pack in a single gzip chunk.')
+                        help='The number of lines to pack in a single gzip/zstd chunk.')
```

Comment:
```suggestion
                        help='The number of lines to pack into a single chunk.')
```

Also, please double-check that the unit is lines, and not bytes. It seems like part of the code may use both.